packages/ListBox/src/hooks/hooks.js

```diff
@@ -77,7 +77,9 @@ export function useAdjustWidth() {
     }
 
     const handleResize = () => {
-      dispatch({ type: useListBox.types.setTriggerWidth, payload: $triggerContainer.offsetWidth });
+      if ($triggerContainer) {
+        dispatch({ type: useListBox.types.setTriggerWidth, payload: $triggerContainer.offsetWidth });
+      }
```

Comment:
move lines 80-82 to a function? The same thing being called on line 75

Reply:
won't do, duplication sometimes is not necessarily bad in my opinion.